spec/support/factory_girl.rb

```diff
@@ -11,10 +11,10 @@
     sequence(:name) { |n| "Item #{n}" }
     image 'https://www.image.png'
     description 'item description'
-    purchase_url 'https://www.purchase.com'
+    category_id 1
```

Comment:
For factories you can just add the line `category` to associate it with your category factory.